src/installer.ts

```diff
@@ -60,7 +61,27 @@ async function promptToReloadWindow() {
   }
 }
 
-async function handleSuccess() {
+async function setBinaryPaths(openUri: Uri) {
+  const homeDir = process.env.HOME;
+  if (!homeDir) {
+    window.showErrorMessage(
+      `Could not determine your home directory. ` +
+      `Set 'lsp.binary' and 'terminal.binary' to the full path` +
+      `where imandrax-cli has been installed:\n` +
+      `[Workspace Settings](${openUri})`
+    );
+    return;
+  }
+
+  const config = workspace.getConfiguration('imandrax');
```

Comment:
This gets the workspace configuration, but the installation is user-wide; we want the user settings here. Except on WSL (or general remotes), where the installation is within the remote, i.e. it needs to go into the remote settings.

Reply:
Done; the secret sauce is that the settings must be `machine-overridable` in `package.json`...